br/pkg/lightning/config/config.go

```diff
@@ -960,7 +960,7 @@ func NewConfig() *Config {
 		TikvImporter: TikvImporter{
 			Backend:                 "",
 			MaxKVPairs:              4096,
-			SendKVPairs:             KVWriteBatchCount,
+			SendKVPairs:             32768,
```

Comment:
do we really need to set it to 32768?

Reply:
old default value, don't take effect now

Reply:
maybe we can delete it